app/src/main/java/com/example/a220402/activity/SignInActivity.kt

```diff
@@ -39,51 +44,86 @@ class SignInActivity : AppCompatActivity() {
         binding = ActivitySignInBinding.inflate(layoutInflater) //inflate는 xml의 뷰를 객체화해준다고 생각하자
         setContentView(binding.root)
 
-        binding.btn.setOnClickListener() {
+        setOnClickListener()
+        initClickEvent()
+        isAutoLogin()
+
+    }
+    private fun setOnClickListener() {
```

Comment:
```suggestion
    private fun setOnLoginBtnClickListener() {
```

메서드명이 조금 더 명확하면 좋을 것 같아요.